css-ui-3/outline-003.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang=en>
+  <meta charset=utf-8>
+  <title>CSS-UI test: outline & layout</title>
+  <link rel="author" href="http://florian.rivoal.net/">
+  <meta name=flags content="">
+  <meta name=assert content="Outline does not influence size or position, and has no impact on layout.">
+  <link rel=help href="https://drafts.csswg.org/css-ui-3/#outline-props">
+  <link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<style>
+div {
+  background: red;
+  width: 100px;
+  height: 100px;
+}
+span:nth-of-type(1) {
+  float: left;
+  width: 30px;
+  height: 30px;
+  outline: solid 10px green;
+  background: green;
+
+  position: relative;
+  top: 10px;
+  left: 10px;
+}
+span:nth-of-type(2) {
+  float: left;
+  width: 0;
+  height: 0;
+  outline: solid 25px green;
```

Comment:
Does this not hit "This specification does not define how multiple overlapping outlines are drawn, or how outlines are drawn for boxes that are partially obscured behind other elements."?